### PR TITLE
Assignment1 fixes

### PR DESCRIPTION
**Pull request by peter-vasut**

### CODE REVIEW

File: assignment1/numpy.ipynb [hunk 1]
```diff
@@ -134,16 +134,16 @@
...
     "print(\"a = {}\".format(a))\n",
     "\n",
-    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14, 15]\n",
+    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14]\n",
     "v = np.arange(10, 15)\n",
     "print(\"v = {}\".format(v))\n",
     "\n",
     "# and also we can set step size. So to create a vector with elements spaced by 10\n",
-    "# and starts at 10 and ends at 50\n",
+    "# and starts at 10 and ends at 40\n",
```
Comment (mrshu) [inline assignment1/numpy.ipynb]: @peto2006 What do you think about adding the actual array to this line? This line could then read then read 

`# and starts at 10 and ends at 40, that is [10, 20, 30, 40]`

**Review by mrshu — COMMENTED** [review]
Just one question, otherwise this looks good to me.

File: assignment1/numpy.ipynb (hunk 1 above)
Comment (peter-vasut) [reply assignment1/numpy.ipynb]: I changed that line to
```# and starts at 10 and ends at 40, that is [10, 20, 30, 40]```

File: assignment1/numpy.ipynb [hunk 2]
```diff
@@ -134,16 +134,16 @@
...
     "print(\"a = {}\".format(a))\n",
     "\n",
-    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14, 15]\n",
+    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14]\n",
     "v = np.arange(10, 15)\n",
     "print(\"v = {}\".format(v))\n",
     "\n",
     "# and also we can set step size. So to create a vector with elements spaced by 10\n",
-    "# and starts at 10 and ends at 50\n",
+    "# and starts at 10 and ends at 40\n, that is [10, 20, 30, 40]",
```
Comment (mrshu) [inline assignment1/numpy.ipynb:146]: @peto2006 Sadly, this does not seem correct (see the attached image), but I'll merge this PR and fix that separately.

Thanks for your help!
screenshot from 2017-10-07 11-12-20

**Pull request merged**